docs-gen/content/rule_set/data_entry/attributes.md

```diff
@@ -7,17 +7,20 @@ weight: 5
 An attribute is a signal with a default value, specified by
 its ```default``` member.
 
-The value set for an attribute by a vspec file can be read by a
-consumer without the need of having the value sent out by a
-publisher. The attribute, in effect, is configuration data that
-can be specified when the vspec file is installed.
+The value set for an attribute by a vspec file can be read by a consumer
+without the need of having the value sent out by a publisher.
+Attribute values can also be provided by publishers, similar to sensors.
+The latter can be useful for attribute values that are likely to change during the lifetime of the vehicle.
+However, this change should typically not be more than once per ignition cycle,
+or else it should be defined as a sensor instead.
 
```

Comment:
stable in power-cycle

Reply:
Possibly drive-cycle? Let's investigate and agree on the right term.   Keeping it open one more week.